addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4689,6 +4701,9 @@ registry.sizing = SnippetOptionWidget.extend({
                 }, 0);
 
                 self.options.wysiwyg.odooEditor.automaticStepActive('resizing');
+                self.trigger_up("snippet_edition_request", { exec: () => {
+                    self.trigger_up("snippet_option_update", { onSuccess: () => {} });
```

Comment:
The whole point of using `snippet_edition_request` is that you can now properly wait for the UI update, giving something as the `onSuccess` value :wink: